components/create/Collection.vue

```diff
@@ -0,0 +1,349 @@
+const selectBlockchain = ref(chainByPrefix?.value || menus[0].value)
+
+const currentChain = computed(() => {
+  return selectBlockchain.value as string
+})
+
+watchEffect(() => setUrlPrefix(currentChain.value as Prefix))
+
+const isKusama = computed(
+  () => currentChain.value === 'ksm' || currentChain.value === 'rmrk'
+)
+
+const isBasilisk = computed(
+  () => currentChain.value === 'bsx' || currentChain.value === 'snek'
+)
+
+const isAssetHub = computed(
+  () => currentChain.value === 'ahk' || currentChain.value === 'ahp'
+)
+
+const createCollection = async () => {
+  let collection:
+    | CollectionToMintBasilisk
+    | CollectionToMintKusama
+    | CollectionToMintStatmine = {
+    file: logo.value,
+    name: name.value,
+    description: description.value,
+    nftCount: unlimited.value ? 0 : max.value,
+  }
+
+  if (isBasilisk.value) {
+    collection['tags'] = []
+  }
+
+  if (isKusama.value) {
+    collection['symbol'] = symbol.value
+  }
+
+  if (collectionDeposit.value && canDeposit.value === false) {
+    return
+  }
+
+  try {
+    showNotification(
+      `Creating Collection: "${name.value}"`,
+      notificationTypes.info
+    )
+    isLoading.value = true
+
+    await transaction(
+      {
+        interaction: Interaction.MINT,
+        urlPrefix: currentChain.value,
+        collection,
+      },
+      currentChain.value
+    )
+  } catch (error) {
+    showNotification(`[ERR] ${error}`, notificationTypes.warn)
+    console.error(error)
+  }
+}
+
+watchEffect(async () => {
+  collectionDeposit.value = 0
+  collectionDeposit.value = depositAmount[currentChain.value]?.collection || 0
+})
+
+watchEffect(async () => {
+  balance.value = 0
+  depositTokenId.value = 0
+  depositSymbol.value = ''
+
+  const api = await apiInstanceByPrefix(currentChain.value)
+  const currentAddress = accountId.value
+  const chain = CHAINS[currentChain.value]
+  const publicKey = decodeAddress(currentAddress)
+  const prefixAddress = encodeAddress(publicKey, chain.ss58Format)
+  const chainInfo = await api.registry.getChainProperties()
+
+  balance.value = await balanceOf(api, prefixAddress)
+
+  if (isBasilisk.value) {
+    depositTokenId.value = await getAssetIdByAccount(api, accountId.value)
+    depositSymbol.value = (
+      await getAssetMetadataById(api, depositTokenId.value)
+    ).symbol
+
+    if (depositAssetSymbol.value === 'KSM') {
+      balance.value = (
+        (await api.query.tokens.accounts(
+          prefixAddress,
+          getKusamaAssetId(currentChain.value)
+        )) as PalletBalancesAccountData
+      ).free.toString()
+    }
+  } else {
+    depositSymbol.value = chainInfo?.tokenSymbol?.toHuman()?.[0]
+  }
+
+  balance.value = format(balance.value, chain.tokenDecimals, false)
+})
+</script>
+
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+
+.o-field:not(:last-child) {
+  margin-bottom: 2rem;
+}
+
+.file {
+  margin-bottom: 0;
+}
+
```

Comment:
```suggestion
```

Look like it has not been used. Or we could use `mb-0`

Reply:
removed